templates/_layout.html

```diff
@@ -3,6 +3,7 @@
 <head>
   <link rel="preconnect" href="https://www.google-analytics.com">
   <link rel="preconnect" href="https://assets.ubuntu.com">
+  <link rel="preconnect" href="https://munchkin.marketo.net">
```

Comment:
What about adding crazyegg here too?

Reply:
Lighthouse didn't recommend preconnecting Crazyegg so it must already be performant enough not to warrant it. To many preconnects can be counter-intuitive.